src/argilla/server/daos/models/records.py

```diff
@@ -66,11 +64,11 @@ class BaseRecordInDB(GenericModel, Generic[AnnotationDB]):
     )
     annotation: Optional[AnnotationDB] = None
 
-    embeddings: Optional[Dict[str, BaseEmbeddingVectorDB]] = Field(
+    vectors: Optional[Dict[str, List[float]]] = Field(
         None,
-        description="Provide the embedding info as a list of key - value dictionary."
-        "The dictionary contains the dimension and dimension sized embedding float list"
-        "Using this way you can skip passing the agent inside of the embedding",
+        description="Provide the vector information as a list of key - value dictionary."
+        "The dictionary contains the dimension and dimension sized float list"
+        "Using this way you can skip passing the agent inside of the vector",
     )
```

Comment:
I could not figure out how to convert the client model to backend model, 

in client we have

vectors = {"my_bert": [....]}

and in backend we defined VectorDB, which will in turn provide

vectors: {"my_bert": {"value": [...]}}

Reply:
@frascuchon can you check if we need such client to backend transformation and point out how to do ?

Reply:
We've already those kind of transformations for predictions and/or annotations. Client and backed has different schemas. So, the SDK classes apply the transformations: 

See [here](https://github.com/argilla-io/argilla/blob/2ff3b959d69b8212b18b50ea66f02465fa66d63b/src/argilla/client/sdk/text_classification/models.py#L56) and [here](https://github.com/argilla-io/argilla/blob/2ff3b959d69b8212b18b50ea66f02465fa66d63b/src/argilla/client/sdk/text_classification/models.py#L99) for two examples